tests/test_dxss.py

```diff
@@ -13,52 +13,26 @@ def test_dummy():
 
 def test_pardiso():
     """Test for PyPardiso. Print info for `pytest -s` logging."""
-    from dxss.solve_1d import PySolver, pypardiso
+    import dxss._solver_backend
 
-    if pypardiso is not None:
+    if dxss._solver_backend.pypardiso is not None:
         import pypardiso as _  # noqa: F401
 
         print("PyPardiso is available.")
     else:
         print("PyPardiso is not available.")
-        with pytest.warns(UserWarning):
-            PySolver(Asp=None, psolver=None)
 
 
-def test_mock_no_pypardiso_for_solve_1d(mocker):
+def test_mock_no_pypardiso_for_solver_warns(mocker):
     # mock no pypardiso installed (even on systems where it's installed)
     mocker.patch.dict(sys.modules, {"pypardiso": None})
-    importlib.reload(sys.modules["dxss.solve_1d"])
 
-    # check that trying to import raises an ImportError
-    with pytest.raises(ImportError):
-        import pypardiso
+    # need to reload the modules in case they where cached
```

Comment:
```suggestion
    # need to reload the modules in case they were cached
```
I _think_ though I also get 'where' and 'were' mixed up so I could well be wrong 😅